src/__tests__/patients/search/NoPatientsExist.test.tsx

```diff
@@ -1,33 +1,24 @@
-import { Icon, Typography, Button } from '@hospitalrun/components'
-import { mount } from 'enzyme'
+import { render as rtlRender, screen } from '@testing-library/react'
 import React from 'react'
 
 import NoPatientsExist from '../../../patients/search/NoPatientsExist'
 
 describe('NoPatientsExist', () => {
-  const setup = () => mount(<NoPatientsExist />)
+  const render = () => rtlRender(<NoPatientsExist />)
 
   it('should render an icon and a button with typography', () => {
-    const wrapper = setup()
+    render()
```

Comment:
For the one time this is is called, I'd just inline it:

```suggestion
import { render, screen } from '@testing-library/react'
import React from 'react'

import NoPatientsExist from '../../../patients/search/NoPatientsExist'

describe('NoPatientsExist', () => {
  it('should render an icon and a button with typography', () => {
    render(<NoPatientsExist />)
```

Reply:
This makes sense. Making this change